index.html

```diff
@@ -170,6 +170,46 @@ <h4 class="supporting-text">
                 <hr>
             </div>
         </section>
+        
+        <section class="contact-form-section">
+            <div class="works-header">
+                <div class="title">
+                    <h1 class="primary-text">
+                        Contact Us
+                    </h1>
+                    <p>
+                        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+                    </p>
+                </div>
+            </div>
+            <div class="contact-form">
+                <form action="https://formspree.io/f/moqroezo" method="post">
+                    <div class="form-group">
+                        <input name="Name" id="name" type="text" placeholder="Enter your name" maxlength="30" required>
+                    </div>
+                    <div class="form-group">
+                        <input name="Email" id="email" type="email" placeholder="Enter your email" required>
+                    </div>
+                    <div class="form-group">
+                        <textarea name="Message" id="Message" rows="5" placeholder="Write your message here" required maxlength="500"></textarea>
+                    </div>
+                    <div class="form-group">
+                        <button type="submit">Get in touch</button>
```

Comment:
- [ ] [OPTIONAL] Make sure you implement the button interactions such as hover and active (when it's pressed)
Please style it following the Figma template 👇 
![Screenshot from 2022-07-06 14-22-26](https://user-images.githubusercontent.com/87965534/177539629-98517da1-d1cb-4896-81fb-69efbfbd6431.png)